.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: ruby
 rvm:
+  - 1.9.3
   - 2.1.0
-
+gemfiles:
```

Comment:
@atz I think it might be singular, i.e. `gemfile:`. Cf. http://docs.travis-ci.com/user/build-configuration/#The-Build-Matrix